runtime/runtime/config/config.go

```diff
@@ -58,6 +59,11 @@ type Runtime struct {
 	TraceEndpoint string          `json:"trace_endpoint"`
 	AuthKeys      []EncoreAuthKey `json:"auth_keys"`
 	SQLDatabases  []*SQLDatabase  `json:"sql_databases"`
+
+	// ShutdownTimeout is the duration before non-graceful shutdown is initiated,
+	// meaning connections are closed even if outstanding requests are still in flight.
+	// If zero, it shuts down immediately.
+	ShutdownTimeout time.Duration `json:"shutdown_timeout"`
```

Comment:
Are we defaulting to zero?

Reply:
Currently yes, will add code on the platform side to specify a default. Originally I had a default of 10s if this is zero, but then you can't actually specify a timeout of zero (unless we change the field to be a `*time.Duration` and use `nil` to represent the default of 10s).